src/test/java/gmbs/model/car/CarTest.java

```diff
@@ -0,0 +1,78 @@
+package gmbs.model.car;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+
+    private static final Car STANDARD_CAR = Car.ofTest("myCar", 5);
+    private static final Car LOWER_POSITION_THAN_STANDARD = Car.ofTest("test1", 1);
+    private static final Car HIGHER_POSITION_THAN_STANDARD = Car.ofTest("test2", 10);
+    private static final Car SAME_POSITION_WITH_STANDARD = Car.ofTest("test3", 5);
+
+    @DisplayName("자동차의 이름값을 가져온다")
+    @Test
+    void getNameValue() {
+        // when
+        String actual = STANDARD_CAR.getNameValue();
+
+        // then
+        assertThat(actual).isEqualTo("myCar");
+    }
+
+    @DisplayName("자동차의 위치값을 가져온다")
+    @Test
+    void getPositionValue() {
+        // when
+        int actual = STANDARD_CAR.getPositionValue();
+
+        // then
+        assertThat(actual).isEqualTo(5);
+    }
+
+    @DisplayName("movable 에 따라 자동차의 전진 여부가 결정된다")
```

Comment:
좀 더 구체적으로 상황을 명시해주는게 좋을 것 같습니다. 
```suggestion
    @DisplayName("movable 가 true인 경우  ~~, false 인 경우 ~~")
```